nss-day-cohort-48/rare-client-boastful-bulldogs
# Create New Tags

## Steps to Test

1. `git fetch --all`
2. `git checkout gqg-tags-create`
3. Open in Chrome, serve using `npm start` on client
4. Verify that `/tags` has a button that allows users to create new tags on the DOM
!!! No server side changes this time!
